### PR TITLE
Create headless service for StatefulSet

### PR DESCRIPTION
**Pull request by aerokite**
Issue: #39

### CODE REVIEW

File: pkg/controller/lib.go [hunk 1]
```diff
@@ -361,3 +361,38 @@ func (c *Controller) CheckDatabaseRestoreJob(
...
+	if found {
+		return nil
+	}
+
+	service := &kapi.Service{
+		ObjectMeta: kapi.ObjectMeta{
+			Name: name,
+		},
+		Spec: kapi.ServiceSpec{
+			ClusterIP: kapi.ClusterIPNone,
```
Comment (TamalSaha) [inline pkg/controller/lib.go]: Don't need this.

You should set `  type: ClusterIP`

Comment (aerokite) [inline pkg/controller/lib.go]: https://kubernetes.io/docs/concepts/services-networking/service/#headless-services

you can create “headless” services by specifying "None" for the cluster IP

Comment (TamalSaha) [inline pkg/controller/lib.go]: Oh. Yeah. But set the type.

Comment (aerokite) [inline pkg/controller/lib.go]: Fixed

**Pull request merged**